graphql_compiler/schema_generation/graphql_schema.py

```diff
@@ -320,18 +324,21 @@ def get_graphql_schema_from_schema_graph(schema_graph, class_to_field_type_overr
     for non_graph_cls_name in sorted(schema_graph.non_graph_class_names):
         if non_graph_cls_name in hidden_classes:
             continue
+        if not schema_graph.get_element_by_class_name(non_graph_cls_name).abstract:
```

Comment:
Merged this code from other branch. I added an implicit reference test. So the test that test that the string representation of the autogenerated schema is equal to SCHEMA_TEXT, implicitly tests that UniquelyIdentifiable is included in the schema.